package.json

```diff
@@ -137,13 +137,12 @@
     "@replayio/cypress": "^1.7.11",
     "@sinonjs/fake-timers": "^9.1.2",
     "@storybook/addon-a11y": "^6.5.16",
-    "@storybook/addon-actions": "6.5.15",
-    "@storybook/addon-essentials": "6.5.15",
-    "@storybook/addon-links": "6.5.15",
+    "@storybook/addon-actions": "6.5.16",
+    "@storybook/addon-essentials": "6.5.16",
+    "@storybook/addon-links": "6.5.16",
     "@storybook/blocks": "^7.0.18",
-    "@storybook/builder-webpack5": "6.5.15",
-    "@storybook/client-api": "6.5.15",
```

Comment:
client-api is removed as it's not used directly and it's a depedency of another package